App.tsx

```diff
@@ -1,14 +1,9 @@
-import { StatusBar } from "expo-status-bar";
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { StyleSheet } from "react-native";
+import MainView from "./components/MainView";
 
 export default function App() {
-	return (
-		<View style={styles.container}>
-			<Text>Open up App.tsx to start working on your app!</Text>
-			<StatusBar style="auto" />
-		</View>
-	);
+	return <MainView></MainView>;
```

Comment:
Why isn't this just `return <MainView/>`? Is there a reason why we'd want a closing tag?